Hazel/Dtls/DtlsUnityConnection.cs

```diff
@@ -207,7 +219,7 @@ protected override void RestartConnection()
             {
                 this.ResetConnectionState();
                 this.nextEpoch.ClientRandom.FillWithRandom(this.random);
-                this.SendClientHello();
+                this.SendClientHello(false);
```

Comment:
The review of my other PR reminded me about labeling inputs---perhaps do that here and below? Not an issue to leave out though

Reply:
Yeah, I'll do it where there are no nearby context clues. Lone bools are a great spot for it.

Reply:
Sounds good!